perun-ldapc/src/main/java/cz/metacentrum/perun/ldapc/processor/impl/EventDispatcherImpl.java

```diff
@@ -308,9 +308,17 @@ protected MessageBeans resolveMessage(String msg, Integer idOfMessage) throws In
 
 		//Debug information to check parsing of message.
 		MessageBeans beans = new MessageBeansImpl();
+
+		boolean shouldntAddVo = false;
+		if (msg.matches("Group:[\\d\\D]*created in Vo:[\\d\\D]*"))
+			shouldntAddVo = true;
+
 		if(!listOfBeans.isEmpty()){
 			int i=0;
 			for(PerunBean p: listOfBeans) {
+				if (shouldntAddVo && p.getBeanName().equals("Vo"))
+					continue;
```

Comment:
I recommend using curly brackets { } or keep it on one line. It's easier to read.